launch.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+args=""
+if [ ! -z "$WAPI_USERNAME" ]; then
+   args=$args" --wapi-username=$WAPI_USERNAME"
+fi
+if [ ! -z "$WAPI_PASSWORD" ]; then
+   args=$args" --wapi-password=$"
```

Comment:
There are a no. of changes like these, which doesn't look right to me...
`args=$args" --wapi-password=$"`

Regardless, since the driver is loading config from environment variables, is there still any reason why we need `launch.sh`?